zerver/webhooks/opbeat/new_error.json

```diff
@@ -1,108 +0,0 @@
-
```

Comment:
Also, could you please explain why you removed all of these payloads? I would suggest removing these in a separate commit and explaining why you are removing them. :)